django/utils/cache.py

```diff
@@ -58,12 +59,11 @@ def dictvalue(t):
             return t[0]
         else:
             return '%s=%s' % (t[0], t[1])
-
+    cc = MultiValueDict()
     if response.get('Cache-Control'):
-        cc = cc_delim_re.split(response['Cache-Control'])
-        cc = dict(dictitem(el) for el in cc)
-    else:
-        cc = {}
+        fields = cc_delim_re.split(response['Cache-Control'])
+        for field in fields:
+            cc.appendlist(*dictitem(field))
```

Comment:
Multiple values should be allowed only for the `no-cache` directive, this change allows for multiple values for all keys what is not desirable.